### PR TITLE
Support VMI/VM Ready conditions 

### PR DESCRIPTION
**Pull request by shiywang**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubevirt/kubevirt/issues/1454

**Special notes for your reviewer**:

**Release note**:

```release-note
Support VMI Ready conditions 
```

### CODE REVIEW

**Comment by fabiand** [comment]
ping?

What's the status?

**Comment by fabiand** [comment]
Btw having a ready condition on VM sounds also good!

**Comment by fabiand** [comment]
ci test please

**Comment by shiywang** [comment]
@fabiand I haven't checked ci failures, but it's ready for review I guess.

File: pkg/virt-controller/watch/vmi.go [hunk 1]
```diff
@@ -332,7 +331,23 @@ func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pods []
 		if !podExists {
 			controller.RemoveFinalizer(vmiCopy, virtv1.VirtualMachineInstanceFinalizer)
 		}
-	case vmi.IsRunning() || vmi.IsScheduled():
+	case vmi.IsRunning():
+		if !conditionManager.HasCondition(vmiCopy, virtv1.VirtualMachineInstanceReady) {
```
Comment (rmohr) [inline pkg/virt-controller/watch/vmi.go]: `virt-handler` needs to set that condition. When the VMI is running, it is cimpletely owned by `virt-handler`.

**Comment by fabiand** [comment]
ping?

**Comment by rmohr** [comment]
Closing this in favour of #1921.

**Pull request closed without merging**